app/views/CancelVATRegistrationPage.scala.html

```diff
@@ -18,7 +18,8 @@
 bullets: components.bullets,
 formHelper: FormWithCSRF,
 button: components.button,
-radios: components.radios)
+radios: components.radios,
+pageHeader: components.pageHeader)
 
```

Comment:
The latest design can be found here: https://design.tax.service.gov.uk/hmrc-design-patterns/page-heading/#options-example-legends-as-page-headings-with-subheading

```
<h2>caption here</h2>
<fieldset>
<legend>
<h1>page heading here</h1>
</legened>
</fieldset>
```

I believe this should be applied on several pages that make use of a fieldset (or radios in this case)